gapic/utils/reserved_names.py

```diff
@@ -12,22 +12,81 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import builtins
-import itertools
-import keyword
```

Comment:
Please could you also check other usages of keywords in the code?

https://github.com/googleapis/gapic-generator-python/blob/13b6c6c1da9b75c171cc53167cd048aba2dc9481/gapic/samplegen/samplegen.py#L44-L47

https://github.com/googleapis/gapic-generator-python/blob/13b6c6c1da9b75c171cc53167cd048aba2dc9481/gapic/schema/api.py#L269-L271

https://github.com/googleapis/gapic-generator-python/blob/13b6c6c1da9b75c171cc53167cd048aba2dc9481/gapic/schema/wrappers.py#L1141-L1146

Reply:
Thanks for the find.

* [gapic-generator-python/gapic/schema/api.py](https://github.com/googleapis/gapic-generator-python/blob/13b6c6c1da9b75c171cc53167cd048aba2dc9481/gapic/schema/api.py#L269-L271)
This is not related to protoplus reserved names. The keyword list is included to avoid syntax errors from Python and "metadata" etc are added because these are "reserved" inside of client methods. For example, "request" is hardcoded in templates as an argument to all client methods.
* [gapic-generator-python/gapic/schema/wrappers.py](https://github.com/googleapis/gapic-generator-python/blob/13b6c6c1da9b75c171cc53167cd048aba2dc9481/gapic/schema/wrappers.py#L1141-L1146) this seems to only deal with Python keywords for the same reason as above. Again, unrelated to protoplus reserved names.

`samplegen` looks legit. Looking into it.

Reply:
`RESERVED_WORDS` in samplegen also doesn't relate to protoplus. Their list is used in https://github.com/googleapis/gapic-generator-python/blob/13b6c6c1da9b75c171cc53167cd048aba2dc9481/gapic/samplegen/samplegen.py#L710-L720, where they try to generate sample code, and want to avoid invalid Python. Actually, their list is also too big (e.g. builtins not need to be there), but it's a separate problem.